framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -154,7 +154,7 @@ qx.Mixin.define("qx.ui.core.MExecutable",
         // remove the old binding
         if (old != null && !old.isDisposed() && ids[property] != null)
         {
-          old.removeBinding(ids[property]);
+          this.removeBinding(ids[property]);
```

Comment:
For the disposal, there might be a problem anyway, because the Button will not be notified when the Command is disposed. You're right in the case of checking. One should change the check above and remove the binding in case of an disposed *old*.

But that doesn't help if you dispose the Command and don't tell it to the Button.